requirements.txt

```diff
@@ -37,6 +37,4 @@ unidecode==1.0.23
 xmljson==0.2.0
 flask
 matplotlib==3.1.1
-torch==1.2.0
-
-#stanford_corenlp==3.9.2
+#torch==1.2.0
```

Comment:
I assume the direct `pip install` in the `Dockerfile` means this is unnecessary? If so, let's delete this line instead of leaving it commented.

Reply:
Just updated, thank you!